### ISSUE TITLE
multiple IPS with samba share and samba nas

### ISSUE THREAD

**Issue by hiagocosta** [dianlight/hassio-addons#77]
When I use the NAS on serving my Ethernet IP Adress, I got to the same folder that I got on the wireless IP adress, but the content is different.

this should behave that way is because i'm using both samba share addon and samba NAS?

can I just use one?

**Comment by hiagocosta** [comment]
I forgot to say that wlan and eth is a different networks

**Comment by dianlight** [comment]
If you see different content the folder differs. 
The addon is created as substitute of the origianl addon so unpredictable results can be right.

HassOS don’t has official support to multiple interfaces and also use it as gateway between two different lan is a borderline use.

I sugget to use only one addon and choose one. Consider that my addon is only for and HassOS installation.The target is to workaournd HassOS external disks limitation, If you have another type of installation is unusefull.

If you like support please post your config detail, addon’s logs and config.

L.

**Comment by hiagocosta** [comment]
I run a live linux on hassos and I saw that it gave me two places, in different partitions, and I fized by deleting one.

the funny is, that one folder is inside my hassos drive, and another is on external device.

thank you for answering and keep up whit the good work!

one more thing, on the mqtt sensors, it doesnt indicates the external device, just inside. it shows unknown (translated)
![image](https://user-images.githubusercontent.com/22087481/163757382-499dfefc-0efc-4632-a9b0-b25901b760e7.png)
